markdown/developers/tutorials/nf_core_contributing_overview.md

```diff
@@ -255,6 +255,25 @@ It will check:
 
 > Tip: when using a code editor, some code editor settings might help you solve any code linting errors by linting when saving a file and even fixing the errors directly. You can check out [this documentation](https://nf-co.re/docs/contributing/editor_plugins) on code editor plugins.
 
+#### Prettier
+
+Prettier is a tool used to make the code more consistent and readable.
+It's a optionated code formatter that enforces a consistent style by parsing the code and re-printing it in its own rules.
+
+Every code and file of the nf-core community needs to be checked and formatted by Prettier before the upload.
+
+The command to check if the style of all files in your directory and its subdirectories is valid for Prettier is the following:
+
+```bash
+prettier -c .
+```
+
+While the command to automatically format the style of all the files in your directory with Prettier is the following:
+
+```bash
+prettier -w .
+```
+
```

Comment:
```suggestion
// TODO: mention pre-commit here after we add it to the template.
```

Reply:
can you be more precise? do you want me to add another paragraph explaining pre-commit?

Reply:
@mashehu what did you meant here?

Reply:
Is resolved now